test/e2e/persistent_volumes.go

```diff
@@ -637,7 +623,56 @@ var _ = framework.KubeDescribe("PersistentVolumes", func() {
 				numPVs, numPVCs := 4, 2
 				pvols, claims = createPVsPVCs(numPVs, numPVCs, c, ns, pvConfig)
 				waitAndVerifyBinds(c, ns, pvols, claims, true)
-				completeMultiTest(f, c, ns, pvols, claims)
+				completeMultiTest(f, c, ns, pvols, claims, v1.VolumeReleased)
+			})
+		})
+
+		// This Context isolates and tests the "Recycle" reclaim behavior.  On deprecation of the
+		// Recycler, this entire context can be removed without affecting the test suite or leaving behind
+		// dead code.
+		Context("when invoking the Recycle reclaim policy", func() {
+			var pv *v1.PersistentVolume
+			var pvc *v1.PersistentVolumeClaim
+
+			BeforeEach(func() {
+				pvConfig.reclaimPolicy = v1.PersistentVolumeReclaimRecycle
+				pv, pvc = createPVPVC(c, pvConfig, ns, false)
+				waitOnPVandPVC(c, ns, pv, pvc)
+			})
+
+			AfterEach(func() {
+				framework.Logf("AfterEach: Cleaning up test resources.")
+				pvPvcCleanup(c, ns, pv, pvc)
+			})
+
+			// This It() tests a scenario where a PV is written to by a Pod, recycled, then the volume checked
+			// for files. If files are found, the checking Pod fails, failing the test.  Otherwise, the pod
+			// (and test) succeed.
+			It("should test that a PV becomes Available and is clean after the PVC is deleted. [Volume][Serial][Flaky]", func() {
+				By("Writing to the volume.")
+				pod := makeWritePod(ns, pvc.Name)
```

Comment:
no, but we can write a couple new functions to mirror the pattern of makeWritePod wrapping makePod.  Alternatively, we could use createClientPod then podExec to make a file on the volume